create_infotexts.py

```diff
@@ -381,9 +374,114 @@ def process_depicted_place(self, place_string, places_mapping):
             self.meta_cats.append("Media_contributed_by_SMVK_without_mapped_place_value")
             place_as_wikitext = place_string
 
-
         self.data["depicted_place"] = place_as_wikitext
 
+    def generate_list_of_stripped_keywords(self, keyword_string):
+        """
+        Transform string of keywords from column <Nyckelord> to list of keywords.
+
+        :param keyword_string: String from column <Nyckelord.
+        :return: list of keywords.
+        """
+        keywords_list = keyword_string.split(", ")
+        if "Svenska Cypernexpeditionen" in keywords_list:
+            keywords_list.remove("Svenska Cypernexpeditionen")
+
+        self.data["Nyckelord"] = keywords_list
+
+    @staticmethod
+    def remove_svenska_cypernexpedition_from_description(description):
+        """
+        Remove the string "Svenska Cypernexpeditionen" from description, since it's in all descriptions.
+
+        :param description: string containg the field value for column <Beskrivning> plus potential enrichments.
+        :return: String with "Svenska Cypernexpeditionen" removed.
+        """
+        new_string = re.sub(" Svenska Cypernexpeditionen\.?", "", description)
+
+        return new_string
+
+    def enrich_description_field(self, item):
+        """
+        Try to add keywords and regional information to description.
+
+        :param item: dictionary containing metadata for one image.
+        :return: string representing altered or unaltered description.
+        """
+        description = self.remove_svenska_cypernexpedition_from_description(item["Beskrivning"])
+
+        if not description.endswith("."):
+            description += "."
+
+        # step 1 in enrichment process
```

Comment:
For step 1 and 2 I would not modify the description string in the other functions. Instead make them return a string to be appended to the description string here.